.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+| Question                                  | Answer
+| ---------------------------------------- | ------------------
+| **JIRA issue**                          | [EZP-XXXXX](https://issues.ibexa.co/browse/EZP-XXXXX)
```

Comment:
```suggestion
| **JIRA issue**                          | [IBX-XXXXX](https://issues.ibexa.co/browse/IBX-XXXXX)
```
or
```suggestion
| **JIRA issue**                          | IBX-XXXX
```
if proper autolinking is configured for the repo

Reply:
Original PR had the previous template. Fixed it in 5f395472